sampleapplication/src/main/java/com/evervault/sampleapplication/ui/views/component/SupportingText.kt

```diff
@@ -0,0 +1,13 @@
+package com.evervault.sampleapplication.ui.views.component
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.text.font.FontStyle
+
+@Composable
+internal fun SupportingText(text: String) {
```

Comment:
Simple component to wrap the supporting text (below the title in the `ListItem`) to make the `NavigationGraph` easier to read and maintain